vng_api_common/schema.py

```diff
@@ -391,6 +391,19 @@ def get_error_codes(self):
         status_codes = sorted({e.status_code for e in exception_klasses})
         return status_codes
 
+    def _get_serializer_field_meta(self, field, direction):
+        meta = super()._get_serializer_field_meta(field, direction)
+        title = meta.get("title", "")
+        if not title:
+            if hasattr(field.parent, "Meta"):
+                if hasattr(field.parent.Meta, "model"):
+                    source = getattr(field.parent.Meta.model, str(field.source), None)
+                    if getattr(source, "field", None):
```

Comment:
In de regel hieronder check je ook op `source`, deze regel lijkt mij ook niet nodig